markov/experimental.R

```diff
@@ -0,0 +1,431 @@
+# Collecting experimental results
+
+library(singlearity)
+suppressPackageStartupMessages(library(tidyverse))
+suppressPackageStartupMessages(library(baseballr))
+suppressPackageStartupMessages(library(lubridate))
+
+# Assumes user is in the overarching "Singlearity" directory 
+
+source(file = "markov/markov.R")
+source(file = "R/get_singlearity_client.R")
+source(file = "examples/pa_pred_simple.R")
```

Comment:
Don't include the examples files in other files.  I had done it before but I think it is bad practice as we want to keep the example functions simple.  It should be easy to copy the actual code if necessary,